roles/scylla-collectd-client/tasks/main.yaml

```diff
@@ -1,5 +1,17 @@
+- name: Gather facts
+  action: ec2_facts
+
+- set_fact: node="scylla-{{ansible_hostname }}"
```

Comment:
There no problem using in EC2 information, like the name (DB), server (scylla), user (tzach), external and internal IP in any combination.

For example, we can use:
"DB-scylla-tzach-external-ip"